plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -320,6 +328,7 @@ function OPDSBrowser:fetchFeed(item_url, username, password, method)
         -- the payload of the request. We'll add that to a table below
         -- and return that as the result of this function.
         local xml = table.concat(sink)
+        logger.dbg("Request returned the following body: ",xml)
```

Comment:
dbg or verbose, or simply leftover and should be removed?

Also space after comma.

Reply:
Leftover. I can take that out.